primitives/src/currency.rs

```diff
@@ -187,6 +187,7 @@ create_currency_id! {
 		LDOT("Liquid DOT", 10) = 3,
 		// 20 - 39: External tokens (e.g. bridged)
 		RENBTC("Ren Protocol BTC", 8) = 20,
+		BNC("Bifrost Native Token", 12) = 133,
```

Comment:
`133` is reserved for Karura & Kusama native tokens. `168 - 255` is available to parachain tokens, plus pls move this line to the right section.